app/components/views/SettingsPage/MiscSettings.js

```diff
@@ -1,6 +1,7 @@
 import { FormattedMessage as T } from "react-intl";
 import { SettingsInput, NumericInput } from "inputs";
 import { InfoDocFieldModalButton } from "buttons";
+import modalStyle from "../../modals/Modals.module.css";
```

Comment:
same here, but as i'm already refactoring this part please leave a TODO and I could handle it later